frontend/e2e/$lang+/_public/apply/adult-child/adult-child-flow.spec.ts

```diff
@@ -0,0 +1,415 @@
+  await page.getByRole('button', { name: 'Continue' }).click();
+
+  await hasError(page, 'Select which provincial or territorial program you are covered under');
+  await page.getByRole('group', { name: 'Provincial or territorial benefits' }).getByRole('radio', { name: 'Alberta Adult Health Benefits' }).check();
+  await page.getByRole('button', { name: 'Continue' }).click();
+}
+
+test.describe('Family flow', () => {
+  test.beforeEach('Navigate to online application', async ({ page }) => {
+    test.setTimeout(60000);
+    await page.goto('/en/apply');
+    await expect(page).toHaveURL(/\/en\/apply\/[a-f0-9-]+\/terms-and-conditions/);
+    await expect(page.getByRole('heading', { level: 1, name: 'Terms and conditions' })).toBeVisible();
+    await page.getByRole('button', { name: 'Agree and continue' }).click();
+  });
+
+  test('Should complete adult and child application', async ({ page }) => {
+    await test.step('Should navigate to type of application page', async () => {
+      await typeOfApplication(page);
+    });
+
+    await test.step('Should navigate to tax filing page', async () => {
+      await taxFiling(page);
+    });
+
+    await test.step('Shoud navigate to file your taxes page', async () => {
+      await fileTaxes(page);
+    });
+
+    // Applicant age is 65 or older, child is under 18
+    await test.step('Should navigate to date of birth page', async () => {
+      await expect(page).toHaveURL(/\/en\/apply\/[a-f0-9-]+\/adult-child\/date-of-birth/);
+      await expect(page.getByRole('heading', { level: 1, name: 'Age' })).toBeVisible();
+      await page.getByRole('button', { name: 'Continue' }).click();
+
+      // check empty fields
+      await hasError(page, 'Date of birth must include a year');
+      await hasError(page, 'Date of birth must include a month');
+      await hasError(page, 'Date of birth must include a day');
+      await hasError(page, 'Select wether or not all the children are under 18');
+
+      await fillOutDOB(page, '1958', '01', '01', 'Yes');
+      await page.getByRole('button', { name: 'Continue' }).click();
+    });
+
+    await test.step('Should navigate to applicant information page when applicant is 18-64', async () => {
+      await expect(page).toHaveURL(/\/en\/apply\/[a-f0-9-]+\/adult-child\/applicant-information/);
+      await expect(page.getByRole('heading', { level: 1, name: 'Applicant information' })).toBeVisible();
+
+      // Back to date of birth
+      await page.getByRole('link', { name: 'Back' }).click();
+      await expect(page).toHaveURL(/\/en\/apply\/[a-f0-9-]+\/adult-child\/date-of-birth/);
+      await expect(page.getByRole('heading', { level: 1, name: 'Age' })).toBeVisible();
+
+      await fillOutDOB(page, '1985', '02', '10', 'Yes');
+      await page.getByRole('button', { name: 'Continue' }).click();
+    });
```

Comment:
I think this test is a bit misleading. The test name indicates that we should navigate to `/applicant-information` if the applicant is between 18-64. The test does navigate to `/applicant-information` but only because in the previous test step, the applicant fills out their date of birth resulting in an age of 66.